packages/amplify-codegen-e2e-core/package.json

```diff
@@ -32,7 +32,7 @@
     "jest-circus": "^27.5.1",
     "jest-environment-node": "^26.6.2",
     "lodash": "^4.17.19",
-    "node-pty": "beta",
+    "node-pty": "1.0.0",
```

Comment:
This might not be necessary for this change. I tried this when troubleshooting various issues. However, this is probably something we should do regardless so that our version is stable. I can try removing this and adding it to a separate change if it is desired.